http4s/src/test/scala/geotrellis/server/http4s/maml/GeneratedTileNode.scala

```diff
@@ -0,0 +1,51 @@
+package geotrellis.server.http4s.maml
+
+import geotrellis.server.core.cog.CogUtils
+
+import com.azavea.maml.ast.{Expression, Literal, MamlKind, RasterLit}
+import com.azavea.maml.eval.tile._
+import io.circe._
+import io.circe.generic.semiauto._
+import cats.effect._
+import cats.syntax.all._
+import geotrellis.raster._
+import geotrellis.vector.Extent
+
+import java.net.URI
+
+
+case class GeneratedTileNode(mapper: Int => Int, size: Int = 256)
+
+//object GeneratedTileNode {
```

Comment:
Hm there are lots of commented out code lines, do you want to include them into master branch? I know it's in a very R&D phase but still.

Reply:
Yeah, this is WIP - not going to merge in comments that aren't meant to be there